### PR TITLE
BL-932-form-position-updates

### PR DESCRIPTION
**Pull request by andrewgomez1992**
## Description

Fixes issues with modal styling interfering with other modals. We fixed this by removing profileSettings.css(which was targeting media queries through Ant-D classNames only). Converted Memo.css to less file to nest all styles in main className. Converting Memo.css to less file auto-fixed radio button being off centered. Changes were motivated by our FE team.

Fixes # (BL-932)

## Loom Video

https://www.loom.com/share/3902288382074348b3bdaeb732d836fd

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by stevenhastings — DISMISSED** [review]
Finally! Trimmed, fit and agile. Looks ready to go. Well, done following through on this ongoing issue.

**Review by ezra-casas — APPROVED** [review]
You gotta love CSS!
Great work Andrew and glad to see this problem solved, hopefully getting another ticket up for the other thing mentioned will get things moving to use the antdesign !

**Review by mcfrr3 — CHANGES_REQUESTED** [review]
Awesome job moving the styles to less! I think this is a great direction since we already have less imported into the project we can make use of its great features 😄. I would just be careful about declaring any styling on the component level if we're making use of stylesheets.

**Review by jack-semder — DISMISSED** [review]
Very nice style changes, and coming from a DS perspective this code is understandable and easy to read. Well done!

File: src/components/pages/Profile/RenderProfileContainer.js
Comment (andrewgomez1992) [reply src/components/pages/Profile/RenderProfileContainer.js:5]: Gotcha will do!

File: src/components/pages/Profile/RenderProfileContainer.js
Comment (andrewgomez1992) [reply src/components/pages/Profile/RenderProfileContainer.js:44]: Would love to pair up to discuss more!

**Review by mcfrr3 — APPROVED** [review]
Everything looks great! Great component-scoped styling, making use of less and top-level class names to make it easier to read and apply specific styles. Great work!

**Review by Charles-Merrill-Johnson — APPROVED** [review]
You all did a great pin-pointing the issue and isolating and implementing the fix. Maneuvering around global styling issues is a challenge of its own!

**Review by stevenhastings — DISMISSED** [review]
Nicely done Andrew! Very comprehendible even from a Data Science perspective. I have been following you on this issue for a while and I'm glad to see you finally wrapping it up.

**Review by theecolya — COMMENTED** [review]
Hey guys long time no see!!! Your css looks great!

**Review by farhaannishtar — DISMISSED** [review]
Good work solving the global styling issues in this ticket. Much needed.

**Review by Nicholas-R-Payne — APPROVED** [review]
Great job updating the CSS to fix this bug!

**Review by Nicholas-R-Payne — APPROVED** [review]
Just resubmitting my review, great job on this again!

**Review by Charles-Merrill-Johnson — APPROVED** [review]
Re approving after removal of code added from merge conflict

**Review by farhaannishtar — APPROVED** [review]

File: src/components/pages/Memos/Memo.less
Comment (ryan-hamblin) [inline src/components/pages/Memos/Memo.less:2]: I like we chose to go with less. I'm a big fan. However, I'm curious about the choice given that our global stylesheet in `/styles/styles.css` is a `.css` file? Doesn't this go against our current pattern?

File: src/styles/styles.css
Comment (ryan-hamblin) [inline src/styles/styles.css:78]: It looks like there are merge conflicts in this file. Just pull/push changes and resolve the merge conflicts and we're good to go.

**Review by ryan-hamblin — CHANGES_REQUESTED** [review]
Excellent work here. It looks like we're fixing a problem that was causing overriding styles on our "Modal" components from "antD". I like this path more than the previous obviously because you're customizing this single component and not overriding the base .less styles that ship with AntD's platform. 

I did want to ask a question to further understand the motivation behind switching to .less in lieu of .css.

Lastly there are merge conflicts blocking this PR inside the `styles.css` to address and this thing is golden.

File: src/components/pages/Memos/Memo.less
Comment (andrewgomez1992) [reply src/components/pages/Memos/Memo.less:2]: I think converting our css files to less files will help any further conflicts like this from happening. I'd love to hear your thoughts.

File: src/styles/styles.css
Comment (andrewgomez1992) [reply src/styles/styles.css:78]: Got with you on this, should be resolved!

File: src/components/pages/Memos/Memo.less
Comment (ryan-hamblin) [reply src/components/pages/Memos/Memo.less:2]: I went and resolved the merged conflicts inline 💯

File: src/styles/styles.css
Comment (ryan-hamblin) [reply src/styles/styles.css:78]: Resolved it.

**Comment by Austin-T-Johnson** [comment]
Great work! The CSS clean up you did looks great and I think it was a good idea to implement Less into the project as nesting is a lot cleaner. Great job showing that the memos page also will need another ticket to add mobile friendly styling.

**Pull request merged**
